recipe/meta.yaml

```diff
@@ -38,6 +41,9 @@ requirements:
     - eigen
     - hdf5
     - openblas
+    # Drop `libcxx` when the upstream issue is fixed
+    # xref: https://github.com/bacpop/pp-sketchlib/issues/92
+    - libcxx       # [osx]
```

Comment:
Filed issue ( https://github.com/conda-forge/pp-sketchlib-feedstock/issues/42 ) to track reverting this once the upstream issue ( https://github.com/bacpop/pp-sketchlib/issues/92 ) is resolved